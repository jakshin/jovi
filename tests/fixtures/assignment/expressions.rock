Put 1 plus 2 minus 3 times 4 into the beat
Say the beat

Put "toil" and "effort" into Work
Say Work

Freedom takes Work
My freedom is everything to me
Give back my freedom

Put Freedom taking Work into the trash
Say the trash
