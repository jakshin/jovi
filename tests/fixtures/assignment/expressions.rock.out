-9
effort
22
